### DISCUSSION TITLE
Editing m3u files

### DISCUSSION THREAD

**Discussion by njack21** [iptv-org/community#1126] [Q&A]
I'm trying out a couple online m3u editors to create my own playlist, m3u4u and m3u.in

I'm importing the full index from https://iptv-org.github.io/iptv/index.m3u, which based on the csv of the database, should include ~38k channels.

Each editor is only displaying a subset of the channels:
* m3u.in displays 11212
* m3u4u displays 9992

This results in being unable to locate many of the channels I want for my curated list. Am I missing something?

**Comment by kennethcochran** [comment]
I'm not familiar with m3u.in but m3u4u.com states in its documentation that playlists are limited to 10000 channels.

https://wiki.m3u4u.com/

If you need to work with more channels than that you'll have to split them into multiple playlists.

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/blob/master/FAQ.md#why-is-the-channel-on-the-iptv-orggithubio-but-not-in-the-playlist
